presto-docs/src/main/sphinx/functions/noisy.rst

```diff
@@ -193,6 +193,124 @@ is supported via the Sketch-Flip-Merge (SFM) data sketch [Hehir2023]_.
         FROM my_table
 
 
+Approximate Percentile Sketching
+---------------------------------------
+
+Noisy approximate percentile sketching (analogous to the deterministic :doc:`q-digest`)
+is supported via the ``QuantileTree`` data sketch to enable differential privacy (DP) for percentile queries.
+The implementation is based on [Qaradji2013]_, [Cormode2011]_, and [Cormode2019]_.
+
+.. function:: noisy_approx_percentile_qtree(x, percentage, epsilon, delta, lower, upper) -> double
+
+    Returns noisy estimates of percentiles from a multiset of values in ``x`` using ``QuantileTree`` private sketches.
+    The value of ``percentage`` must be between zero and one and must be constant for all input rows.
```

Comment:
```suggestion
    The value of ``percentage`` must be between zero and one, and must be constant for all input rows.
```
Nit: suggest add a comma as a pause in the sentence, for readability to separate "must be constant for all input rows" as a separate topic.